src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -315,6 +337,7 @@ def main():
             input_shapes=input_shapes,
         )
     _LOGGER.info(model)
+    num_streams = parse_num_streams(model, scenario)
```

Comment:
this will fail if you ask the ORT engine for num_streams, it doesn't have that property. either need to add it to `ort_engine.py` or do some thing else